contrib/enum_metrics/test/test_clear_errant_enums.py

```diff
@@ -0,0 +1,27 @@
+from unittest import TestCase
```

Comment:
I am exploring that currently. I thought I will create this PR, get everyone's opinion to see if I am going in the right direction. If it works out okay, I will submit it in my subsequent PR. I am also looking at somehting similar for elastic search.